src/components/App/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,233 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`COMPONENTS - App renders self and correct routes 1`] = `
+<App>
+  <section>
+    <styled.h1>
+      <StyledComponent
+        forwardedComponent={
+          Object {
+            "$$typeof": Symbol(react.forward_ref),
+            "attrs": Array [],
+            "componentStyle": ComponentStyle {
+              "componentId": "sc-bdVaJa",
+              "isStatic": true,
+              "lastClassName": "iulWLw",
+              "rules": Array [
+                "
+  font-size: 2rem;
```

Comment:
I think I forgot to add *jest-styled-components* (https://github.com/styled-components/jest-styled-components). Will make snapshots a bit more readable.